src/GeositeFramework/js/Plugin.js

```diff
@@ -401,6 +405,10 @@ require(['use!Geosite',
             view.$uiContainer = $uiContainer;
 
             $uiContainer
+                // Call `hibernate` method on plugin when the eye icon is clicked
+                .find('.plugin-eye').on('click', function () {
+                    model.hibernate();
+                }).end()
```

Comment:
Sounds like a regression defect then. For example, if you add some layers using the Map Layers plugin in [production](http://maps.coastalresilience.org/gulfmex/), then minimize the plugin, the layers remain on the map.

Basically, we just need to keep the existing behavior, but replace the minimize icon with an eye ball.

Just pointing this out to ensure that we keep things as backwards-compatible as possible for existing plugins.